test/max.test.js

```diff
@@ -0,0 +1,8 @@
+import test from 'ava'
+import max from '../src/max'
```

Comment:
This should be: `import {max} from '../src'` (see the other tests).
This means that you'll need to import `max` into the `src/index.js` and add it to the list of exports. Could you do that update please?